include/open62541pp/types.hpp

```diff
@@ -1090,6 +1184,32 @@ class Variant : public TypeWrapper<UA_Variant, UA_TYPES_VARIANT> {
         return getArrayCopyImpl<T>();
     }
 
+    // Unified API for setting a value.
+    // A template parameter is used to select the variant
+    // policy instead of using additional functions with "Copy" suffix.
+    // Default policy is Copy for more robust behavior (avoid potential dangling references and
```

Comment:
What's your thought on using "Copy" as default policy when setting values?
On one hand it is safer due to accidental dangling references; on the other hand I imagine the `Variant` class would be used as a short-living wrapper to encapsulate some data for sending it away in most cases.

Reply:
Copy is the safe choice, therefore it should be the default in my opinion.
I agree, that the `Variant` is usually a short-living object, therefore it makes sense to just borrow the data. But this is an optimization that should be introduced explicitly by the library users.

After some experience with the current policy model, I would like to simplify this. Either the `Variant` owns the data or it doesn't. The `ReferenceIfPossible` policy hides if the provided value is actually borrowed or copied. If a value can not be borrowed, a compile error is the safest choice and makes clear, that an explicit copy is needed.

Do you agree?

Reply:
Yes I agree, it would be better to drop `ReferenceIfPossible` 👍

Reply:
Removed `ReferenceIfPossible`